code/modules/mob/living/simple_animal/borer/borer.dm

```diff
@@ -16,7 +16,7 @@
 	wander = 0
 	pass_flags = PASSTABLE
 	universal_understand = 1
-	holder_type = /obj/item/weapon/holder/borer
+	//holder_type = /obj/item/weapon/holder/borer //Theres no inhand sprites for holding borers, it turns you into a pink square
```

Comment:
we can make borers pickuppable easily if someone makes sprites for them. We don't have those sprites atm